File: a1-programming-basics-javascript/02b-conditional-statements-exercise/06-world-swimming-record.js
function worldSwimmingRecord(input) {
  let record = Number(input[0]);
  let distance = Number(input[1]);
  let time = Number(input[2]);

  waterResistance = Math.floor(distance / 15) * 12.5;
  newRecord = distance * time + waterResistance;

  absoluteTime = Math.abs(record - newRecord);

  if (newRecord < record) {
    console.log(
      `Yes, he succeeded! The new world record is ${newRecord.toFixed(
        2
      )} seconds.`
    );
  } else {
    console.log(
      `No, he failed! He was ${absoluteTime.toFixed(2)} seconds slower.`
    );
  }
}

File: b1-softuni-programming-basics-python/02b-conditional-statements-exercise/06_world_swimming_record.py
current_record = float(input())
distance = float(input())
time_per_m = float(input())
delay = distance // 15 * 12.5

new_record = distance * time_per_m + delay

if new_record < current_record:
    print(f"Yes, he succeeded! The new world record is {new_record:.2f} seconds.")

else:
    print(f"No, he failed! He was {new_record-current_record:.2f} seconds slower.")
